pkgs/tools/networking/p2p/jesec-rtorrent/rtorrent.nix

```diff
@@ -6,8 +6,10 @@
 , gtest
 , libtorrent
 , ncurses
-, jsonRpcSupport ? true, nlohmann_json
```

Comment:
There is no reason to split this line.

The reason for this is obvious: the first is a Boolean argument that controls the insertion of the second one as a dependency.

Reply:
I agree, except for the fact that this was the result of running the file through nixpkgs-fmt -- no formatter will be able to know this (and if Nixpkgs is auto-formatted in the future, this will happen anyways).

Reply:
This was the only thing the formatter found? Six lines in a diff that worsens the legibility of a code?
And people ask why I don't like them...

Also formatters are not to be the last word. Machines are meant to help us, not us to help them.